src/mono/browser/runtime/startup.ts

```diff
@@ -664,8 +665,8 @@ export function mono_wasm_set_main_args(name: string, allRuntimeArguments: strin
 /// 3. At the point when this executes there is no pthread assigned to the worker yet.
 export async function configureWorkerStartup(module: DotnetModuleInternal): Promise<void> {
     initWorkerThreadEvents();
-    currentWorkerThreadEvents.addEventListener(dotnetPthreadCreated, (ev) => {
-        mono_log_debug("pthread created 0x" + ev.pthread_self.pthreadId.toString(16));
+    currentWorkerThreadEvents.addEventListener(dotnetPthreadCreated, () => {
+        // mono_log_debug("pthread created 0x" + ev.pthread_self.pthreadId.toString(16));
```

Comment:
should it stay commented?

Reply:
yes, it was noisy, but it's good to keep it there so that we know it could be done.